core/templates/dev/head/i18n.js

```diff
@@ -51,7 +51,8 @@ oppia.controller('I18nFooter', [
   // The $timeout seems to be necessary for the dropdown to show anything
   // at the outset, if the default language is not English.
   $timeout(function() {
-    $scope.currentLanguageCode = $translate.use();
+    $scope.currentLanguageCode = $translate.use() ||
```

Comment:
(I suggest also including the reference link to http://stackoverflow.com/a/28903658 )